### ISSUE TITLE
Unable to remove MDN Popups

### ISSUE THREAD

**Issue by robbpage** [microsoft/vscode#216278]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.90.0
- OS Version: Windows 11 (22631.3737)

I have unchecked every box in the settings and verified in the settings.json. These obnoxious and utterly pointless pop-ups should not STILL be showing up and yet here we are. Years after people have been complaining about the ability to remove this useless feature.

PLEASE fix this. This should be an EXTREMELY simple fix for a problem that shouldn't even exist in the first place since literally NO ONE uses these obnoxious popups.

**Comment by robbpage** [comment]
I highly doubt a .1 update will fix the issue I've been having for years but just to be a good sport I just updated, and to my utter shock...

the problem persists. as i knew it would.

**Comment by robbpage** [comment]
is there a department at microsoft we can BEG to? i will do it. seriously. this is such an obnoxious and utterly indefensible "feature" to have in your software that it almost HAS to be intentional. No one uses this "feature" and there are documented complaints about this going back YEARS... so why is it still a problem?

**Comment by gjsjohnmurray** [comment]
Are the settings mentioned in https://github.com/microsoft/vscode/issues/97979#issuecomment-736094869 not working for you?

**Comment by robbpage** [comment]
> Are the settings mentioned in [#97979 (comment)](https://github.com/microsoft/vscode/issues/97979#issuecomment-736094869) not working for you?

nope. I've tried every possible combination I can find on the internet and nothing removes them.

**Comment by gjsjohnmurray** [comment]
Please be specific about the context in which they are appearing despite those settings. What language mode is the editor? What contents? Where do they appear? Does it happen after you run the command to reload with all extensions disabled? Or with a new empty profile with only those settings added?

**Comment by Karliz24** [comment]
@VSCodeTriageBot

**Comment by robbpage** [comment]
> Please be specific about the context in which they are appearing despite those settings. What language mode is the editor? What contents? Where do they appear? Does it happen after you run the command to reload with all extensions disabled? Or with a new empty profile with only those settings added?

the popups appear no matter what i set. i have literally tried EVERYTHING. i have removed all extensions. i have tried every checkbox in the main and workspace settings. i have tried every trick that the internet has, even the ones that Bard and Claude invented to try and help. nothing removes them.

if there was ANY better alternative (that isn't stupidly overpriced like jetbrains) i would ditch vscode in a second over this pathetic lack of skill in fixing an extremely simple bug but alas, i'm basically stuck dealing with their incompetence.

**Comment by robbpage** [comment]
a complete fresh install. i removed ALL settings and extra folders that the uninstall (for some inane reason) doesn't remove. no extensions installed. default EVERYTHING.

unchecked the 4 settings for MDN hover in the settings menu and BIG SHOCK these obnoxious popups are STILL there.

worthless devs. absolutely utterly worthless. look at their utter silence on this issue after YEARS of complaints. microsoft you need to hire smarter people because this is utterly unacceptable.

but then, windows still won't let you delete an folder if you had it open in the last 24 hours, and people have been complaining about that bug for decades. so it shouldn't be shocking to come across another example of their incompetence.
